### PR TITLE
Update the remove null geometries algorithm (include the remove empty option)

### LINKED ISSUE #5017: Request in QGIS ([needs-docs][processing] Optionally, also remove EMPTY geometries in RemoveNullGeometries algorithm)
### Request for documentation
From pull request QGIS/qgis#35084
Author: @gacarrillor
QGIS version: 3.14

**[needs-docs][processing] Optionally, also remove EMPTY geometries in RemoveNullGeometries algorithm**

## Description

Empty geometries do not contain coordinates. Like `NULL` geometries, empty geometries are not spatial data in a strict sense. 

QGIS still lacks a `Remove EMPTY geometries` algorithm, and to make it easier for users to clean a vector layer, the remove EMPTY geometries could be an option in the existing `Remove null geometries`.

The algorithm dialog looks like this:

image

### Commits tagged with [need-docs] or [FEATURE]

### PR DESCRIPTION
**Pull request by havatv**
Goal:

Ticket(s): Fix #5017

- [ ] Backport to LTR documentation is required

### CODE REVIEW

File: docs/user_manual/processing_algs/qgis/vectorgeometry.rst [hunk 1]
```diff
@@ -4850,12 +4855,17 @@ Parameters
      - ``INPUT``
      - [vector: any]
      - Input vector layer (with non-NULL geometries)
+   * - **Also remove empty geometries**
```
Comment (DelazJ) [inline docs/user_manual/processing_algs/qgis/vectorgeometry.rst]:
```suggestion
   * - **Also remove empty geometries** |314|
```
and in substitutions

**Comment by havatv** [comment]
Travis: ```make: *** No rule to make target 'fasthtml'.  Stop.```

File: docs/user_manual/processing_algs/qgis/vectorgeometry.rst (hunk 1 above)
Comment (havatv) [reply docs/user_manual/processing_algs/qgis/vectorgeometry.rst]: substitutions fixed in a separate commit.

**Comment by DelazJ** [comment]
> Travis: make: *** No rule to make target 'fasthtml'. Stop.

This is due to the branch being a bit old. The latest master does not have fasthtml build in the Makefile so we should simply use html here (fixed in master). Merging Should be harmless on this point.

**Pull request merged**